rockpaperscissor.py

```diff
@@ -27,9 +27,17 @@
 ---.__(___)
 ''')
 
+# set initial value for user_move
+user_move = -1
+
 game_ascii = [Rock, Paper, Scissors]
 
-user_move = int(input("What do you choose for rock 0, for paper 1 for scissors 2.:-"))
+while (user_move < 0) or (user_move > 2):
```

Comment:
if you add if statement here then it will be great

Reply:
what is the if statement needed for? 
the user will be questioned to insert an input, as long as the input is between 0 and 2. 
With the try except block, we check for a ValueError, which would rise, if the user inserts a float number.
if the user does enter a float, a print statement will show up, to please insert a valid input